crev-lib/src/local.rs

```diff
@@ -432,6 +432,35 @@ impl Local {
         Ok(())
     }
 
+    pub fn fetch_all(&self) -> Result<()> {
+        eprintln!("Fetching all crev-proofs repositories ({})",
+                  self.cache_remotes_path().display());
+
+        for entry in fs::read_dir(self.cache_remotes_path())? {
+            let path = entry?.path();
+            if !path.is_dir() {
+                continue
+            }
+
+            let repo = git2::Repository::open(&path);
+            if repo.is_err() {
+                continue
```

Comment:
IMO, it would be better to split the loop body into `fetch_single_repo`, so you can just `?` from it, and ignore it from the place you call it, instead of checking errors manually here.